app/graphql/types/event_type.rb

```diff
@@ -4,6 +4,7 @@ class EventType < Types::BaseObject
     field :user_id, ID, null: false
     field :name, String, null: true
     field :date, String, null: true
+    field :week_number, String, null: true
```

Comment:
Should we change this to an Integer as well?

Reply:
Yes I think so.